.github/workflows/cloud_chatops.yaml

```diff
@@ -80,9 +80,41 @@ jobs:
           context: "{{defaultContext}}:cloud-chatops"
           tags: "harbor.stfc.ac.uk/stfc-cloud-staging/cloud-chatops:${{ steps.commit_sha.outputs.sha_short }}"
 
+
+  check_version_update:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout master
+        uses: actions/checkout@v4
+        with:
+          ref: 'master'
+          path: 'main'
+
+      - name: Checkout current working branch
+        uses: actions/checkout@v4
+        with:
+          path: 'branch'
+
+      - name: Compare versions
+        if: ${{ github.ref != 'refs/heads/master' }}
+        uses: khalford/check-version-action@main
```

Comment:
I think this is using an action you maintain? Any reason we can't use a community action / put the steps here?

The concern is the bus factor - if your account got deleted / removed / there was a security update whilst you were away we're unable to update this

Reply:
Could we instead bring either bring the action into `stfc/` or copy the steps into this workflow then?

Reply:
Yes, a repo will need to be made for the action

Reply:
Repo is made and waiting on inital pull request https://github.com/stfc/check-version-action/pull/1